Passphrase Creation unit tests fail randomly

Sometimes these tests fail:

```
FAILED TESTS:
  Passphrase Creation
    Using laptop
      ✖ sets the next step once the movement is completed
        HeadlessChrome 70.0.3538 (Linux 0.0.0)
      AssertionError: expected 100 to be above 100
          at Context.<anonymous> (webpack:///src/components/passphraseCreation/index.test.js:58:67 <- src/tests.js:391403:84)
```
https://jenkins.lisk.io/job/lisk-hub/job/1.6.0/14/consoleFull

--- a/src/components/passphraseCreation/index.test.js
+++ b/src/components/passphraseCreation/index.test.js
@@ -55,7 +55,7 @@ describe('Passphrase Creation', () => {
       }
 
       wrapper.update();
-      expect(wrapper.find(CreateFirst)).to.have.prop('percentage').greaterThan(100);
+      expect(wrapper.find(CreateFirst)).to.have.prop('percentage').at.least(100);
       expect(wrapper.find(CreateFirst)).to.have.prop('hintTitle', 'by moving your mouse.');
       expect(wrapper.find(CreateFirst)).to.have.prop('address');
       expect(wrapper.find(CreateFirst)).to.have.prop('step', 'info');

